tests/main/interfaces-custom-device-app-slot/task.yaml

```diff
@@ -28,6 +28,8 @@ prepare: |
     snap debug can-manage-refreshes | MATCH false
 
     snap ack "$TESTSLIB/assertions/testrootorg-store.account-key"
+    snap ack "$TESTSLIB/assertions/developer1.account"
+    snap ack "$TESTSLIB/assertions/developer1.account-key"
```

Comment:
How is this related?

Reply:
it never worked, but store-state was running without -e, so it failed quietly on https://github.com/canonical/snapd/pull/14686/files/08eab5ab589736cc661f00cffec08f72bcf1f0a7#diff-234265efe637e5730e201767212cce6ce6f4f5fea39bfb026a01913fa281afccL128